source/community/a-guide-to-the-design-system-monthly-chat.html.md

```diff
@@ -0,0 +1,75 @@
+
+The community manager and delivery managers will then ensure speakers are recruited, and the session is supported and organised.
+
+## How support the session
+There are various roles required in running the monthly chat. They are:
+
+### Facilitator
+
+Introduces the session and general housekeeping
+Introduces speakers
+Facilitates questions for show & tells
+Leads the lean coffee discussion
+
+### Speaker
+
+- delivers a show & tell
+- answers questions regarding their show & tell
+
+### Retro host
+- designs an appropriate retro on Padlet
+- facilitates the retro during the call
+
+### Chat moderator (min. 2 people)
+
+- answers questions in the Zoom chat from attendees
+- shares links from the presentations
+
+### Troubleshooter
+
+- logs into the admin account during the session
+- records the session to their computer
+- enables captions and transcript
+- sets up breakout rooms
+- allocates co-hosts
+
+### Notetaker
+
+- records notes from the playback of the lean coffee discussions
+
+### Screen sharer
+
+- screen shares the presentation
+- allocates co-presenter so others can move slides
+- takes no other active role in the session as this interferes with screen sharing
+
+## How to promote the session
+Comms and tickets for the session should be published at least two weeks in advance of the call. This is to give community members enough notice to attend.
+
+Comms are sent out via the mailing list on Mailchimp, and on our Slack channels. Tickets are created via Eventbrite.
+
+## How to prepare for the session
+Before the event, ensure all volunteer roles are filled. This is the responsibility of the delivery manager assigned to community work.
+
+We should aim to have prepared the slide deck a week before the session so it can be shared with attendees in advance. This is generally best accessibility practice.
+
+If you are due to speak at the event, it is advised that you access Zoom via the desktop app rather than the browser. This is because accessing Zoom through a browser will prevent you from being able to “drive” the slide deck during your presentation. 
+If using the desktop app, ensure the correct permissions are in place on your PC so you’re able to screen share, and share audio and video. You can do this via going to ‘System Settings’ — ‘Privacy & Security’ — ‘Screen and System Audio Recording’ and enabling permission for Zoom. You may need to restart Zoom for this to take effect. Follow the same steps for enabling ‘Microphone’ and ‘Video” within ‘Privacy & Security’.
+
+If speaking, it’s helpful to have a quiet room to present from to limit background noise. It’s also helpful to have your speaker notes in the main slide deck. During the session, the host will be able to give you co-presenter permissions which enables you to view your speaker notes on the same screen as the slide.
```

Comment:
Does this work? "During the session, the host will be able to give you co-presenter permissions which enables you to view your speaker notes on the same screen as the slide."

I couldn't work out how to get speaker notes up when I did my presentation for the first DS Day.

Reply:
It's definitely not obvious!